### ISSUE TITLE
Deprecate or Rename

### ISSUE THREAD

**Issue by jakirkham** [conda-forge/deeplearnjs-feedstock#35]
Upstream renamed this to `tfjs-core`. Also this feedstock hasn't seen any maintenance since last year. Should this be deprecated or renamed?

**Comment by jakirkham** [comment]
cc @conda-forge/core

**Comment by CJ-Wright** [comment]
@conda-forge/deeplearnjs thoughts? It seems that many version bump PRs have not been merged.

**Comment by jakirkham** [comment]
Can we blacklist this feedstock? We appear to still be generating update PRs for it.

**Comment by CJ-Wright** [comment]
Interesting I thought that I had fixed that. Let me try again.

I don't really have a setup for blacklisting beyond archive.

**Comment by jakirkham** [comment]
Oh ok. Should we archive it?
